src/modalities/config/config.py

```diff
@@ -314,6 +314,7 @@ class LLMDataLoaderConfig(BaseModel):
     pin_memory: bool
     shuffle: bool
     skip_num_batches: Optional[int] = 0
+    fixed_num_batches: Optional[int] = None
```

Comment:
I was also thinking about this. I decided against `num_training_steps` because a dataloader does not need to know the concept of a "step", which is more related to optimizers. We could indeed rename it to `num_batches`, I think both are fine.

Reply:
I would also opt for `num_batches `.